src/workerd/api/streams/internal.h

```diff
@@ -147,7 +147,7 @@ class ReadableStreamInternalController: public ReadableStreamController {
 
   Tee tee(jsg::Lock& js) override;
 
-  kj::Maybe<kj::Own<ReadableStreamSource>> removeSource(jsg::Lock& js);
+  kj::Maybe<kj::Own<ReadableStreamSource>> removeSource(jsg::Lock& js, bool ignoreDisturbed=false);
```

Comment:
Not going to block on this nit but generally I'm not a fan of bool flags like this. I'd prefer an `enum class` approach.. like

```
enum class RemoveSourceOption {
  NONE,
  IGNORE_DISTURBED
};

kj::Maybe<kj::Own<ReadableStreamSource>> removeSource(
    jsg::Lock& js,
    RemoveSourceOption option = RemoveSourceOption::NONE);
```